metastats-domain/src/main/groovy/life/qbic/metastats/filter/SequencingModeCalculator.groovy

```diff
@@ -14,33 +12,31 @@ import org.apache.logging.log4j.Logger
  */
 class SequencingModeCalculator {
 
-    private static final Logger LOG = LogManager.getLogger(SequencingModeCalculator.class)
-
     /**
      * Calculates the sequencing mode based on all filenames of a sample.
      * If the files contain R1 and R2 then the mode is paired end. Else it is single end.
      * @param filename a string containing all filenames of a sample comma separated
-     * @return String of determined sequencing mode 
+     * @return String of determined sequencing mode
      */
-    static String calculateSequencingMode(String filename) {
+    static String calculateSequencingMode(String filenames) {
```

Comment:
```suggestion
    static SequencingMode calculateSequencingMode(List<String> filenames) {
```
```suggestion
    static SequencingMode calculateSequencingMode(String filename) {
```
Think about converting this into an enum. According to your JavaDoc, there are only two choices here.

Reply:
I will collect all your refactoring suggestions and state them in an issue

Reply:
#7